engine/crates/validation/tests/validation_errors/input_object_cycles.graphql

```diff
@@ -0,0 +1,21 @@
+input A {
+    otherA: A!
+}
+
+input B {
+  other: [B]
```

Comment:
Worth making this `[B!]!` so you get a proper test of lists that can't hit the field-is-nullable case?

Reply:
Good point — done and pushed